### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vladgusakovskiy**
[DEMO LINK](https://vladgusakovskiy.github.io/react_static-list-of-todos/)

### CODE REVIEW

**Review by OleksiiPustovit — APPROVED** [review]
Good job 🔥

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+  const {
+    user,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <article className={completed ? 'TodoInfo TodoInfo--completed'
+      : 'TodoInfo'}
+    >
```
Comment (AndriiRut) [inline src/components/TodoInfo/TodoInfo.tsx:19]: instal classnames module and use it in classNames

**Review by AndriiRut — CHANGES_REQUESTED** [review]

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+        {
+          'TodoInfo--completed': completed,
+        },
+      )
+    }
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+      {user ? <UserInfo user={user} /> : null}
```
Comment (denys-danyliuk) [inline src/components/TodoInfo/TodoInfo.tsx:30]: let's write some message instead of a null, something like 'There is no user'

**Review by denys-danyliuk — APPROVED** [review]
GJ!

**Pull request closed without merging**